### DISCUSSION TITLE
Filesystem.stat(path) Android doesn't work

### DISCUSSION THREAD

**Discussion by loremru** [capawesome-team/capacitor-plugins#81] [Q&A]
On ios Filesystem.stat works properly. 
On Android `pickVideo` returns path:
`content://com.google.android.apps.photos.contentprovider/-1/2/content://media/external/video/media/1000000018/ORIGINAL/NONE/video/mp4/906508663`
When i use this path with Filesystem.stat() application crashes
I think i should transform `content://` to `file://`. How can I do it?

**Comment by robingenz** [comment]
Thank you for your request. 
I need the exception message to be able to help you.

**Reply by loremru** [reply]
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.String.hashCode()' on a null object reference

**Reply by robingenz** [reply]
You should report this bug to the Capacitor team. This has nothing to do with the FilePicker plugin.

**Comment by loremru** [comment]
There is `size` in `pickVideo` result
